### PR TITLE
Get rid of chef_gem.

### PR DESCRIPTION
**Pull request by luishdez**
Source: https://github.com/edelight/chef-mongodb/pull/370

### CODE REVIEW

**Comment by luishdez** [comment]
Not sure if this is the best idea… but works as a quick fix

**Comment by josephholsten** [comment]
LGTM so long as it gets a comment explaining when we can rip it out

File: recipes/mongos.rb [hunk 1]
```diff
@@ -24,7 +24,15 @@
 node.override['mongodb']['instance_name'] = 'mongos'
 
 include_recipe 'mongodb::install'
-include_recipe 'mongodb::mongo_gem'
+
+ruby_block 'chef_gem_at_converge_time' do·
```
Comment (triccardi-systran) [inline recipes/mongos.rb]: Why is there a middle dot at the end of the line?

Comment (josephholsten) [inline recipes/mongos.rb]: oh, that's strange. this didn't get run through travis. I'll fix the dot.

Comment (josephholsten) [inline recipes/mongos.rb]: fixed in #39, waiting on travis build

**Comment by luishdez** [comment]
Closing in favor of https://github.com/chef-brigade/mongodb-cookbook/pull/39

**Pull request closed without merging**